src/store.js

```diff
@@ -1,14 +1,16 @@
 import { createStore } from 'redux';
 import _ from 'lodash';
 
+const assign = (o1, o2) => _.assign({}, o1, o2);
```

Comment:
yes indeed, and because of the mutation redux thinks that the state hasn't changed